pkg/proto/nbcontract/v1/clusterconfig.proto

```diff
@@ -49,6 +50,34 @@ message ClusterNetworkConfig {
 
   // Route table name
   string route_table = 5;
+
```

Comment:
Although these are networking related, I separated the fields above because they are exclusively referenced in the `azure.json` file. This entire `ClusterConfig` contains fields that are only referenced in `azure.json`. So, in the future if we want to go with Alex's idea of separating the `azure.json` fields into its own JSON blob, i think it might be best to just keep the following fields in a separate `NetworkConfig`.

Reply:
I see I see. wasn't aware of this. Then I agree with you. I will move them back. Thanks